### PR TITLE
shopfloor_mobile: REF index to render app w/ qweb template

### PR DESCRIPTION
**Pull request by simahawk**
Features:

* makes the app odoo-like extendible
* use module version to force reload of resources changing URL
* added icons (to be improved)
* display module version in the app

Ref: 1451

### CODE REVIEW

File: shopfloor_mobile/templates/assets.xml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+            type="module"
+        />
+        <script
+            t-attf-src="/shopfloor_mobile/assets/src/lib/vue-super-call.js?#{app_version}"
+            type="module"
+        />
+        <script
+            t-attf-src="/shopfloor_mobile/assets/src/lib/vuetify/vuetify.js?#{app_version}"
+        />
+        <!-- TODO: make file local -->
```
Comment (sebalix) [inline shopfloor_mobile/templates/assets.xml]: Is it a good opportunity right now?

File: shopfloor_mobile/templates/assets.xml [hunk 2]
```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="shopfloor_app_assets" name="Shopfloor app assets">
+        <link
+            href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900"
```
Comment (sebalix) [inline shopfloor_mobile/templates/assets.xml]: I suppose it's too late to change this dependency to google font Roboto? :)

**Comment by simahawk** [comment]
@sebalix cleaned up external resources.

**Review by sebalix — APPROVED** [review]
:clap:

**Review by guewen — APPROVED** [review]

**Pull request merged**